config/locales/models/activity.en.yml

```diff
@@ -385,6 +392,7 @@ en:
         geography: Will the benefitting recipient be a region or country?
         identifier: Enter your unique identifier
         intended_beneficiaries: What are the intended beneficiaries?
+        is_oda: Is it ODA or Non-ODA?
```

Comment:
Can we make this `"Is the activity ODA or Non-ODA"`? I appreciate we're working from minimal content in the designs, but I think `it` is a bit less specific.